engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -326,7 +326,7 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
       pushFailedJobMetadata(jobKey, None, throwable, retryableFailure = false)
       context.parent ! WorkflowExecutionFailedResponse(stateData.executionStore, stateData.outputStore, List(throwable))
       goto(WorkflowExecutionFailedState) using stateData.mergeExecutionDiff(WorkflowExecutionDiff(Map(jobKey -> ExecutionStatus.Failed)))
-    case Event(SucceededResponse(jobKey, returnCode, callOutputs), stateData) =>
+    case Event(SucceededResponse(jobKey, returnCode, callOutputs, _), stateData) =>
```

Comment:
Does detritus need to be metadata'd too? Probably should be part of `pushSuccessfulJobMetadata` if `returnCode` and `callOutputs` are?